src/pip/_internal/commands/install.py

```diff
@@ -267,6 +268,10 @@ def run(self, options: Values, args: List[str]) -> int:
         if options.use_user_site and options.target_dir is not None:
             raise CommandError("Can not combine '--user' and '--target'")
 
+        if (options.parallel_downloads is not None) and (
+            options.parallel_downloads < 1
+        ):
```

Comment:
```suggestion
        if options.parallel_downloads < 1:
```

The None case shouldn’t be possible anymore right?

Reply:
Yep, I missed that.